docs/connector/spark/hive.rst

```diff
@@ -0,0 +1,109 @@
+.. Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
the copyright you have copied from other rst is malformed by you editor, please send a follow up to fix ,FYI
https://kyuubi.readthedocs.io/en/latest/connector/spark/hive.html

Reply:
Hi @Yikf. It's recommended to add screenshots for manual local doc build if appropriate, and it's better for a committer to ask for it before merging, cc @pan3793